.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+require:
+  - rubocop-performance
```

Comment:
I think since `Rakefile` is in this project - `rubocop-rake` should be included as well